### PR TITLE
fix: make APU bleed PSI XX if ADIRS not aligned

### LINKED ISSUE #1484: APU Bleed Pressure Indication
**Mod Version**

v0.4.0

**Describe the bug**

On the APU SD page, the APU Bleed Pressure is indicated immediately even when the ADIRS is still "OFF"

**To Reproduce**

1. On Ground
2. Energize Aircraft Electrical Network
3. Monitor the APU SD page
4. Switch on the APU Master SW
5. Start the APU
6. Observe APU Bleed Pressure Indication

**Expected behavior**

APU Bleed Pressure on the APU System Display should show "XX PSI" when the ADIRS is still "OFF" and will only indicate the Bleed Pressure when the ADIRS is activated

The APU bleed will still work normally if the indication are amber "xx"

**Actual behavior**

During APU start-up, I noticed that the APU Bleed Pressure is indicated right-away as soon as it's available, contrary
to the actual behaviour on the real aircraft

**References**

apu

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Discord username (if different from GitHub):

avteknisyan

### PR DESCRIPTION
**Pull request by Benjozork**
Fixes #1484 

## Summary of Changes

- APU BLEED PSI now shows "XX" when ADIRS is not aligned

## References

See issue

Discord username (if different from GitHub): someperson#4953

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js [hunk 1]
```diff
@@ -90,27 +100,46 @@ var A320_Neo_LowerECAM_APU;
...
+                    }
+
+                    this.APUBleedPressure.textContent = SimVar.GetSimVarValue("L:APU_BLEED_PRESSURE","PSI");
+                    this.APUBleedPressure.setAttribute("class", "APUGenParamValue");
+                }
 
-                if (SimVar.GetSimVarValue("APU GENERATOR ACTIVE", "Bool") == 1 && SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool") === 0) {
-                    this.APUGenAvailArrow.setAttribute("visibility", "visible");
-                } else {
+                if (APUPctRPM < 95 || !adirsAligned) {
```
Comment (Jimver) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:133]: If it is a float is probably doesn't matter but `APUPctRPM <= 95` might be better considering the other bound is `APUPctRPM > 95`

Comment (Jimver) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_APU.js:133]:
```suggestion
                if (APUPctRPM <= 95 || !adirsAligned) {
```
So like this

**Comment by danbw9** [comment]
Name: danbw9#7028
Date of testing: Current date 22/10/2020
Version of the sim: 1.9.5.0
PR Tested: #1491

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:

Start from C&D and started up APU 
Tested both with no ADIRS in ALIGN and also with ADIRS in ALIGN, alignment not completed. The amber crosses are there on both on Bleed indication on APU SD page.

Overall Rating: Good
Conclusions of the testing: LGTM. Looked into it and it seems that ADIRU 2 is what the APU bleed pressure seems to go by.  ADIRU 2 being supplied by a static source linked to the Bleed pressure indication.

**Comment by Benjozork** [comment]
Merging since this is a simple enough change.

**Pull request merged**